Polygon/include/CGAL/draw_polygon_with_holes_2.h

```diff
@@ -122,7 +129,7 @@ class SimplePolygonWithHoles2ViewerQt : public Basic_viewer_qt
   }
 
 protected:
-  const P2& p2;
+  ;
```

Comment:
remove

Reply:
Sorry about that.  I probably moved the declaration to the argument list of `compute_elements` and left the rest there. :grin: